airbyte-webapp/src/components/YamlEditor/TestBuilderServer.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from "components/ui/Button";
+
+import { useConfig } from "config";
+
+export const TestBuilderServer: React.FC<unknown> = () => {
```

Comment:
We should probably remove this from the PR before merging, since this was just for testing the POC and not actually something we want in the code

Reply:
[done](https://github.com/airbytehq/airbyte/pull/18410/commits/2d91e182862013ef2668068d280608dd78b8e9a7)